test/python/testann.py

```diff
@@ -136,6 +136,14 @@ def testNumPyLegacy(self):
         # Validate count
         self.assertEqual(ann.count(), 100)
 
+    @patch('platform.system')
+    def testmocdarwin(self, mock_platform):
```

Comment:
I'd go with the existing convention (i.e. testFaissDarwin or testFaissMacOS)

Reply:
Updated to testFaissMacOS